ui/dashboards/src/components/Panel/PanelContent.tsx

```diff
@@ -29,7 +29,14 @@ export function PanelContent(props: PanelContentProps) {
   const PanelComponent = plugin?.PanelComponent;
 
   if (isLoading) {
-    return <Skeleton variant="rectangular" width={contentDimensions?.width} height={contentDimensions?.height} />;
+    return (
+      <Skeleton
+        variant="rectangular"
+        width={contentDimensions?.width}
+        height={contentDimensions?.height}
+        aria-label="Loading..."
```

Comment:
Added this both for a11y and to make it easier to select for in tests.

Reply:
Do you mean to have the `...` (doesn't make a big difference, but I noticed the test selector only looks for `Loading`)

Reply:
If I recall correctly, there was one other spot that had a label like this, so I tried to match it. The selector isn't an exact match, so it doesn't need the `...`.